cpanfile

```diff
@@ -77,7 +82,14 @@ if ( "$]" >= 5.020 ) {
     requires 'Dist::Zilla::PluginBundle::Author::OALDERS';
     requires 'Dist::Zilla::PluginBundle::DROLSKY';
     requires 'Dist::Zilla::PluginBundle::Milla';
-    requires 'Dist::Zilla::PluginBundle::RJBS';
+
+    if ( "$>" < 5.026 ) {
```

Comment:
Why is this `$>` rather than `$]`?

Reply:
indeed, using effective user id here is a mistake
please adjust https://perldoc.perl.org/perlvar#$%3E

Reply:
Changed to $]